src/App.js

```diff
@@ -1,12 +1,17 @@
 import './App.css';
+import './Styles/index.css'
 import { Header } from './Components/Header';
 import { About } from './Components/About';
+import { Wish } from './Components/Wish';
+import { Connect } from './Components/Connect';
 
 function App() {
   return (
-    <div>
+    <div className='background'>
```

Comment:
Here, and in all files you have problems with css naming, try this:
https://ru.bem.info/methodology/